var/spack/repos/builtin/packages/nettle/package.py

```diff
@@ -39,3 +40,9 @@ class Nettle(AutotoolsPackage):
 
     depends_on('gmp')
     depends_on('m4', type='build')
+
+    def configure_args(self):
+        if sys.platform == 'darwin':	
```

Comment:
`spec.satisfies('platform=darwin')` would be avoiding the import of `sys` but both variants are currently in use..

Reply:
I stole the pattern from the `gcc` package. 

If I counted right (ran out of fingers and toes, the dog had to help...):

- "import sys\n" + "sys.platform == 'darwin'" is 35 characters; while
- "self.spec.satisfies('platform=darwin')" is 38 characters.

Parsimony suggests that I leave it as is.  It wouldn't take very much for almost any other reason to be better than that though.  I'm happy to change it.